runtime/browser/native_window.cc

```diff
@@ -204,33 +222,32 @@ void NativeWindow::RemoveRotationHandler(int id) {
   handler_table_.erase(id);
 }
 
+NativeWindow::ScreenOrientation NativeWindow::RotationOrientation() const {
```

Comment:
Please split the function and change the name
NativeWindow::ScreenOrientation orientation() const {
    return NativeAngleToOrientation(rotation_, natural_orientation_);
}

static NativeWindow::ScreenOrientation NativeAngleToOrientation(int angle, ScreenOrientation natural_orientation) {
}